dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -1720,3 +1726,54 @@ def test_std_raises_on_index():
         match="`std` is only supported with objects that are Dataframes or Series",
     ):
         dd.from_pandas(pd.DataFrame({"test": [1, 2]}), npartitions=2).index.std()
+
+
+@pytest.mark.skipif(not PANDAS_GT_200, reason="ArrowDtype not supported")
+def test_std_raises_with_arrow_string_ea():
+    pa = pytest.importorskip("pyarrow")
+    ser = pd.Series(["a", "b", "c"], dtype=pd.ArrowDtype(pa.string()))
+    ds = dd.from_pandas(ser, npartitions=1)
```

Comment:
We tend to use multiple partitions in tests to make sure that our blocked algorithms are more thoroughly tested. Sometimes operations will pass with a single partition, but fail with multiple. This isn't actually relevant for this case, because we're raising early. But just wanted to raise this best practice for future reference. 

```suggestion
    ds = dd.from_pandas(ser, npartitions=2)
```